test/utils/runners.go

```diff
@@ -635,6 +635,32 @@ func (*TrivialNodePrepareStrategy) CleanupNode(node *api.Node) *api.Node {
 	return &nodeCopy
 }
 
+type LabelNodePrepareStrategy struct {
+	labelKey   string
+	labelValue string
+}
+
+func NewLabelNodePrepareStrategy(labelKey string, labelValue string) *LabelNodePrepareStrategy {
+	return &LabelNodePrepareStrategy{
+		labelKey:   labelKey,
+		labelValue: labelValue,
+	}
+}
+
+func (s *LabelNodePrepareStrategy) PreparePatch(*api.Node) []byte {
+	labelString := fmt.Sprintf("{\"%v\":\"%v\"}", s.labelKey, s.labelValue)
+	patch := fmt.Sprintf(`{"metadata":{"labels":%v}}`, labelString)
```

Comment:
Overwrite them, but it's either that or failing the test alltogether. I think it's fine, as this is only for testing (either e2e or integration)